apps/web/src/components/Feed/FeedList.tsx

```diff
@@ -59,11 +64,36 @@ export default function FeedList({
     feedEventRefs
   );
 
+  const isMobileOrMobileLargeWindowWidth = useIsMobileOrMobileLargeWindowWidth();
+  const suggestedProfileSectionHeight = useMemo(
+    () => (isMobileOrMobileLargeWindowWidth ? 320 : 360),
+    [isMobileOrMobileLargeWindowWidth]
+  );
```

Comment:
nit: don't need to memoize primitive types; memoization mostly meant for complex types like computed values, arrays, objects. 

can just do:
```js
const suggestedProfileSectionHeight = isMobileOrMobileLargeWindowWidth ? 320 : 360
```